doc/cookbook/custom-properties.md

```diff
@@ -31,14 +31,14 @@ class CustomPropertyListener implements EventSubscriberInterface
     
     public static function getSubscribedEvents()
     {
-        return array(
-            TransformEvent::POST_TRANSFORM => 'addCustomProperty',
-        );
+        return [
+            PostTransformEvent::class => 'addCustomProperty',
```

Comment:
I haven't seen this before, what makes this different from `TransformEvent::POST_TRANSFORM`?

Reply:
That's the new standard for Symfony event dispatcher if I'm not wrong.
Use class name as event name, when it's possible (ie. not same class used for different events).

Reply:
Just found an article that explained the move: https://symfony.com/blog/new-in-symfony-4-3-simpler-event-dispatching

Reply:
By the way in the previous PR adding support for elastica, I removed all these constants and used different classes for each event, so that should now be the only way to listen / subscribe on these events.

Reply:
Thank you.

Reply:
> By the way in the previous PR adding support for elastica, I removed all these constants and used different classes for each event, so that should now be the only way to listen / subscribe on these events.

We should also list those changes in https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/CHANGELOG-6.0.md

Reply:
Yes I see that there is a line about the `FOS\ElasticaBundle\Persister\Event\Events` but more details should probably be added to facilitate the move for other events

Reply:
Done in https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1657